config/locales/en.yml

```diff
@@ -1078,6 +1078,9 @@ en:
         to our <a href="http://www.osmfoundation.org/wiki/License/Takedown_procedure">takedown
         procedure</a> or file directly at our
         <a href="http://dmca.openstreetmap.org/">on-line filing page</a>.
+      trademarks_title_html: <span id="trademarks"></span>Trademarks
+      trademarks_1_html: |
+        OpenStreetMap and the looking glass logo are registered trademarks of the OpenStreetMap Foundation. If you have questions about your use of the marks, please send your questions to the <a href="http://wiki.osmfoundation.org/wiki/Licensing_Working_Group">LWG</a>.
```

Comment:
As the first mention of the Licensing Working Group, it should be left expanded, not abbreviated to LWG.